exporter/clickhouseexporter/README.md

```diff
@@ -214,6 +214,40 @@ WHERE ServiceName = 'clickhouse-exporter'
 Limit 100;
 ```
 
+### Metrics
+
+Metrics data is stored in different clickhouse tables depending on their types. The tables will have a suffix to
+distinguish which type of metrics data is stored.
+
+| Metrics Type          | Metrics Table          |
+| --------------------- | ---------------------- |
+| sum                   | _sum                   |
+| gauge                 | _gauge                 |
+| histogram             | _histogram             |
+| exponential histogram | _exponential_histogram |
+| summary               | _summary               |
+
+Before you make a metrics query, you need to know the type of metric you wish to use. If your metrics come from
+Prometheus(or someone else uses OpenMetrics protocol), you also need to know the
+[compatibility](https://github.com/open-telemetry/opentelemetry-specification/blob/main/specification/compatibility/prometheus_and_openmetrics.md#prometheus-and-openmetrics-compatibility)
+between Prometheus(OpenMetrics) and OTLP Metrics.
```

Comment:
Yes, it's not friendly for users, considering this is designed for storing data in an organized way which means the user query experience is not the primary. I'm not expecting End Users to direct make a query,  I think there needs a query layer to hide those details and give a universal way to query. This needs more discussion and it's out of the exporter's scope.

Reply:
If it was possible to create an issue that can users can track or follow along, either with Clickhouse or in this project. Just so it isn't forgotten or ignored that it can be improved.